plugins/tileacq.py

```diff
@@ -689,4 +689,5 @@ def acquire(self, dlg):
             raise
         finally:
             logging.info("Tiled acquisition ended")
+            main_data.stage.moveAbs(orig_pos)
```

Comment:
The point of adding the move here, is that you can remove it from the exception handlers ;-)